app/screens/home/channel_list/categories_list/threads_button/threads_button.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {TouchableOpacity, View} from 'react-native';
+
+import {switchToGlobalThreads} from '@actions/local/thread';
+import Badge from '@components/badge';
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import {useIsTablet} from '@hooks/device';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'row',
+        paddingLeft: 3,
+    },
+    badge: {
+        backgroundColor: theme.mentionBg,
+        position: 'relative',
+        left: 0,
+    },
+    icon: {
+        color: changeOpacity(theme.sidebarText, 0.5),
+        fontSize: 24,
+    },
+    iconHighlight: {
+        color: theme.sidebarText,
+    },
+    textContainer: {
+        flex: 1,
+    },
+    text: {
+        color: changeOpacity(theme.sidebarText, 0.72),
+        paddingLeft: 9,
+        ...typography('Body', 200, 'Regular'),
+    },
+    textHighlight: {
+        color: theme.sidebarText,
+        ...typography('Body', 200, 'SemiBold'),
+    },
+}));
+
+type Props = {
+    currentChannelId: string;
+    unreadsAndMentions: {
+        unreads: number;
+        mentions: number;
+    };
+};
+
+const ThreadsButton = ({currentChannelId, unreadsAndMentions}: Props) => {
+    const isTablet = useIsTablet();
+    const serverUrl = useServerUrl();
+
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
+
+    const handlePress = useCallback(preventDoubleTap(() => {
+        switchToGlobalThreads(serverUrl);
+    }), [serverUrl]);
+
+    const {unreads, mentions} = unreadsAndMentions;
+
+    const [iconStyle, textStyle] = useMemo(() => {
```

Comment:
this component is missing the active state, see channel_item where it has the border and the background color.

Reply:
can you share a screenshot of the active state after you exported the styles from the channel_item and imported them here..